### PR TITLE
Create user & group on-demand

### PR DESCRIPTION
**Pull request by chrissng**
No need `ONBUILD` anymore

### CODE REVIEW

File: Dockerfile
Comment (guangie88) [inline Dockerfile]: You might want to remove this comment, since it is no longer applicable.

Comment (chrissng) [reply Dockerfile]: oops, got it

**Review by guangie88 — APPROVED** [review]
LGTM, but you would have to wait for @xkjyeah's approval :+1:

**Review by xkjyeah — APPROVED** [review]

**Pull request merged**
